### ISSUE TITLE
In VS2017 Migration, remove global.json

### ISSUE THREAD

**Issue by julielerman** [dotnet/roslyn-project-system#1184]
as per Livar Cunha

**Comment by NickCraver** [comment]
Details from https://github.com/dotnet/cli/issues/5351:

## Steps to reproduce
1. Grab the Dapper repo here: https://github.com/StackExchange/dapper-dot-net/tree/23e6b64c9ff0121ce1c9a9dcf80648be20780d5c
2. Open `Dapper.sln` in VS 2017 RC
3. Upgrade all projects when prompted.

## Expected  behavior
These files should no longer exist in the main or project folders:
- `project.json`
- `project.lock.json`
- `*.xproj`
- `*.xproj.user`
- `global.json`

## Actual behavior
Files are still present, not removed.

## Environment data
`dotnet --info` output:
```
.NET Command Line Tools (1.0.0-rc3-004504)

Product Information:
 Version:            1.0.0-rc3-004504
 Commit SHA-1 hash:  b312fbb176

Runtime Environment:
 OS Name:     Windows
 OS Version:  6.3.9600
 OS Platform: Windows
 RID:         win81-x64
 Base Path:   C:\Program Files\dotnet\sdk\1.0.0-rc3-004504
```

**Comment by 333fred** [comment]
In order to maintain parity with CLI, we need to do the following steps:
* [ ] Look in the solution directory for a global.json
* [ ] Set the cwd for migration to be the solution directory

**Comment by gulbanana** [comment]
Wouldn't removing global.json leave no way to specify the sdk version for people with multiple CLI installs?

**Comment by 333fred** [comment]
The global.json is now deleted, but now removed from the solution, so there's a broken link in the explorer. I'm leaving this open to track that issue.

**Comment by 333fred** [comment]
Also, @NickCraver, the other files besides the global.json (.xproj, xproj.user, project.json, project.lock.json) should have been removed, so I'd guess that your binaries were out of date for that.

**Comment by julielerman** [comment]
@333fred  .. just for clarification: "global.json is noW deleted" or "noT deleted"?

**Comment by 333fred** [comment]
It is deleted, but not currently removed from the solution, so a broken link will appear. That will be fixed soon.

**Comment by 333fred** [comment]
Fixed with #1287.

**Comment by 333fred** [comment]
Updated the title for what is broken in RC3
